src/Detector/LowCardinalityIndexDetector.php

```
@@ -31,33 +32,47 @@ public function detect(array $explainResult): bool
+                if ($this->checkTable($table)) {
                     return true;
                 }
             }
         }
 
         // Check ordering_operation
-        if (isset($queryBlock['ordering_operation']['table']) && $this->checkTable($queryBlock['ordering_operation']['table'])) {
+        if (
+            isset($queryBlock['ordering_operation']['table']) &&
+            is_array($queryBlock['ordering_operation']['table']) &&
+            $this->checkTable($queryBlock['ordering_operation']['table'])
```

PHPStan (2.1.33)

Parameter `#1` $table of method Koriym\SqlQuality\Detector\LowCardinalityIndexDetector::checkTable() expects array{table_name: string, access_type: string, possible_keys?: string|null, key?: string|null, rows: int, rows_examined_per_scan: int, filtered: float, attached_condition?: string, ...}, array<mixed, mixed> given.
Offset 'used_columns' (list<string>) does not accept type mixed: mixed is not a list.

(argument.type)